
float64[3] pos
float64[3] dpos
float64[3] pos_d
float64[3] dpos_d
float64[3] ddpos_d

float64[3] pos_error
float64[3] ori_error

float64 F_d
float64 F_I
float64[6] F_ext0
float64[6] F_extK
float64 pos_a
float64 dpos_a
float64 ddpos_a
float64 originDepthDistance
float64 contactDepthDistance
float64 filterDepthDistance
float64 VirtualForce
float64 filterVirtualForce
